From d4af0809eb876d2c1df2cfeda42e32562aef8139 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Mon, 30 Jul 2018 00:29:16 -0700
Subject: [PATCH 19/20] core: Add bootimage only cmdline flag

Sometimes we want to include cmdline options only for the bootimage,
not for the recovery image, so add a new flag for it

Change-Id: I05107536a75150b1f5ca40aa39499cfe011b6139
---
 core/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/core/Makefile b/core/Makefile
index 177c5926a..b0e157b72 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -663,8 +663,9 @@ endif
 endif
 
 INTERNAL_KERNEL_CMDLINE := $(strip $(BOARD_KERNEL_CMDLINE) buildvariant=$(TARGET_BUILD_VARIANT) $(VERITY_KEYID))
+INTERNAL_BOOTIMAGE_CMDLINE := $(strip $(INTERNAL_KERNEL_CMDLINE) $(BOARD_BOOTIMAGE_CMDLINE))
 ifdef INTERNAL_KERNEL_CMDLINE
-INTERNAL_BOOTIMAGE_ARGS += --cmdline "$(INTERNAL_KERNEL_CMDLINE)"
+INTERNAL_BOOTIMAGE_ARGS += --cmdline "$(INTERNAL_BOOTIMAGE_CMDLINE)"
 endif
 
 INTERNAL_MKBOOTIMG_VERSION_ARGS := \
-- 
2.17.1

